.github/workflows/test.yaml

```diff
@@ -179,7 +202,7 @@ jobs:
     - name: Run the checks
       run: |
         python --version
-        tox --parallel -e lint,apidocs,narrativedocs,mypy
```

Comment:
I see that there is a separate job for `narrativedocs` above now.  I like this.  CI jobs that generate 10,000 lines of output that are a mix of 5 different logical jobs are miserable to interact with.  I hope that `lint` and `mypy` will be similarly split apart someday.

I don't see a separate job for `apidocs` being created.  Maybe this is because the API docs are now generated exclusively by readthedocs?

Reply:
Yes. apidocs is built as part of read the docs.

At the same time, if `apidocs` build fails, the CI is still green.

I am working to fix apidocs as part of https://github.com/twisted/twisted/pull/11578

-----------

the reasoning for my change was that since GHA VMs have 2CPU.. there is not much use to run more than 2 jobs.

----------

And yes. At the same time, reading the output is hard.


But I will refactor this to run in a matrix to reduce the need for copy/paste inside the YAML file.